### PR TITLE
Consolidate find stop times for stop query

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
Removes the shorter query for operators, as it was causing issues with the schedule in the nearby view.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by josh-willis-arcadis — APPROVED** [review]
LGTM!

**Comment by josh-willis-arcadis** [comment]
looks like snapshots need to be cleaned up

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Still seeing a bug. If I go to the schedule view for a stop, then click on a stop popup on the map, I see the weird "Loading Stop...". 

We might need a new approach to getting these icons?

File: lib/components/map/default-map.tsx [hunk 1]
```diff
@@ -86,6 +86,8 @@ function getLayerName(overlay, config, intl) {
       return intl.formatMessage({ id: 'components.MapLayers.streets' })
     case 'Satellite':
       return intl.formatMessage({ id: 'components.MapLayers.satellite' })
+    case 'Stops':
```
Comment (miles-grant-ibigroup) [inline lib/components/map/default-map.tsx:89]: Why is this here?

**Review by miles-grant-ibigroup — APPROVED** [review]
I'm so confused how/why this works but it totally does! Yay

File: lib/components/map/default-map.tsx (hunk 1 above)
Comment (amy-corson-ibigroup) [reply lib/components/map/default-map.tsx:89]: Because the otpUi-stopsandstations layer has "Stops" as the name!
image

**Pull request merged**